### PR TITLE
ARTEMIS-904 Remove cyclic dependencies from artemis-cli

### PR DESCRIPTION
**Pull request by bennetelli**
move classes and methods to their correct location to avoid cyclic dependencies between packages and classes. I also removed unused methods.

### CODE REVIEW

File: artemis-cli/src/main/java/org/apache/activemq/artemis/util/ServerUtil.java [hunk 1]
```diff
@@ -121,18 +121,6 @@ public static int getServer(Connection connection) {
...
-         ClientSession session = ((ActiveMQConnection) connection).getInitialSession();
-         TransportConfiguration transportConfiguration = session.getSessionFactory().getConnectorConfiguration();
-         String port = (String) transportConfiguration.getParams().get("port");
-         if (Integer.valueOf(port) == server + 61616) {
-            return connection;
-         }
-      }
-      return null;
-   }
-
```
Comment (jbertram) [inline artemis-cli/src/main/java/org/apache/activemq/artemis/util/ServerUtil.java]: This method is used in the "clustered-static-oneway" example.

**Comment by jbertram** [comment]
Please squash the commits. Also, keep an eye on formatting. It looks like a few things got changed in the last commit that seem incorrect.

**Comment by bennetelli** [comment]
squashed.

**Comment by jbertram** [comment]
Looks like you may need to rebase.

**Comment by jbertram** [comment]
I rebased this and merged it.

**Comment by clebertsuconic** [comment]
@jbertram / @bennetelli nothing works after this.. you can't even start a server.

What was changed here that broker it? I may have to revert it if not fixed by tomorrow. i have a release to make.

**Comment by jbertram** [comment]
I don't understand. What specifically isn't working now? I ran the PR build myself which starts the broker quite a few times for different tests and everything worked.

**Comment by clebertsuconic** [comment]
Try running a server... it won't work... 

one way to check is to run ./artemis-distribution/src/test/scripts/validate-spaces.sh

**Pull request closed without merging**